public/js/cita.json

```diff
@@ -1,345 +1,345 @@
 [
   {
-    "title": "Citizens Advice",
-    "address": "26 Spring Gardens, BUXTON, Derbyshire, SK17 6DE",
-    "phone": "0808 146 7709",
-    "lat": 53.2588544,
-    "lng": -1.9120955
+    "title": "Allerdale Citizens Advice",
+    "address": "Vulcans Lane, WORKINGTON, Cumbria, CA14 2BT",
+    "phone": "01900 68981",
+    "lat": 54.6444994,
+    "lng": -3.5488266
   },
   {
-    "title": "Citizens Advice",
-    "address": "3rd Floor, 60 Charles Street, LEICESTER, Leicestershire, LE1 1FB",
-    "phone": "0116 326 6326",
-    "lat": 52.6351881,
-    "lng": -1.1295463
+    "title": "Arun and Chichester Citizens Advice",
+    "address": "Town Hall, Clarence Road, Bognor Regis, West Sussex, PO21 1LD",
+    "phone": "01243 860516",
+    "lat": 50.7831729,
+    "lng": -0.6703726
   },
   {
-    "title": "Citizens Advice",
-    "address": "Beaumont Lodge, Beaumont Fee, LINCOLN, Lincolnshire, LN1 1UL",
-    "phone": "01522 828617",
-    "lat": 53.2311876,
-    "lng": -0.5421441
+    "title": "Bournemouth Citizens Advice",
+    "address": "West Wing, Town Hall, Bourne Avenue, BOURNEMOUTH, Dorset, BH2 6DX",
+    "phone": "01202 203661",
+    "lat": 50.7220011,
+    "lng": -1.8833211
   },
   {
-    "title": "Citizens Advice",
+    "title": "Caerphilly Citizens Advice",
+    "address": "2B De Clare House, 5 Alfred Owen Way, Pontygwindy Industrial Estate, CAERPHILLY, Caerphilly, CF83 3HU",
+    "phone": "02920 855222",
+    "lat": 51.5892056,
+    "lng": -3.2267737
+  },
+  {
+    "title": "Cardiff and Vale Citizens Advice",
+    "address": "119 Broad Street, BARRY, Vale of Glamorgan, CF62 7TZ.",
+    "phone": "01446 704998",
+    "lat": 51.4032521,
+    "lng": -3.273753
+  },
+  {
+    "title": "Central and East Northamptonshire Citizens Advice",
     "address": "Town Centre House, 7/8 Mercers Row, NORTHAMPTON, Northamptonshire, NN1 2QL",
     "phone": "0300 3239940",
     "lat": 52.2375028,
     "lng": -0.8962066999999999
   },
   {
-    "title": "Citizens Advice",
-    "address": "Shelfanger Road, DISS, Norfolk, IP22 4EH",
-    "phone": "01379 658205",
-    "lat": 52.3795521,
-    "lng": 1.1064057
-  },
-  {
-    "title": "Citizens Advice",
-    "address": "16-17 St Mark's Street, PETERBOROUGH, Cambridgeshire, PE1 2TU",
-    "phone": "01733 887922",
-    "lat": 52.5792817,
-    "lng": -0.2414297
+    "title": "Ceredigion Citizens Advice",
+    "address": "Napier Street, CARDIGAN, Ceredigion, SA43 1ED",
+    "phone": "01239 613 707",
+    "lat": 52.0850404,
+    "lng": -4.657687399999999
   },
   {
-    "title": "Citizens Advice",
-    "address": "Swingate House, Danestrete, STEVENAGE, Hertfordshire, SG1 1AF",
-    "phone": "01438 721760",
-    "lat": 51.90273999999999,
-    "lng": -0.2037096
+    "title": "Chelmsford Citizens Advice",
+    "address": "Burgess Well House, Coval Lane, CHELMSFORD, Essex, CM1 1FW",
+    "phone": "01245 205567",
+    "lat": null,
```

Comment:
Good spot, it's an ambiguous address in Google Maps, I'll wing in the lat/lng manually